cli/node.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func MakeNodeCommand() *cobra.Command {
```

Comment:
todo: I think it's confusing to have both `node` and `client` subcommands that do the same thing. If the naming of `node` is preferred over `client` we should just rename it and merge them.

Reply:
I'm also not a big fan of having a dedicated `node` command just for one subcommand. But I think moving it under the `client` might send a wrong message to the users as there might be multiple clients dealing with the same node and command `client identity` is even more confusing.
Maybe we should discuss it on the standup.